src/longopt/grdinfo_inc.h

```diff
@@ -25,41 +25,35 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
+	{ 0, 'C', "oneliner",
+	          "n,t",               "numeric,name_at_end",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
+	{ 0, 'D', "tiles",
 	          "",                  "",
+	          "i",                 "ignore_empty",
 		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
+	{ 0, 'E', "extreme|extrema",
+	          "x,y",               "x,y",
+	          "l,L,u,U",           "min,minpos,max,maxneg",
 		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'F', "geographic",        "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'G', "force_remote",      "", "", "", "", GMT_TP_STANDARD },
```

Comment:
Then I think `force_download` is clearer.

Reply:
yes, but I think it can  just be --download.  Saying --download obviously means do the download, hence forcing.